script1.sh

```diff
@@ -0,0 +1,11 @@
+echo "Dvir Avraham" > ~/output.txt
+printf "DvirAvraham" > ~/filename.txt
+mkdir ~/new
+cp ~/filename.txt ~/new/
+mv ~/new/filename.txt ~/new/Dvir_Avraham.txt
+gedit ~/new/your_name.txt
```

Comment:
To avoid opening gedit interactively, you can use a command-line text editor such as nano to edit the file and add your name and favorite animal.